### PR TITLE
add usa-layout-grid to sass entrypoint

### PR DESCRIPTION
**Pull request by jasonfu9966**
[USWDS Version `3.7.1`](https://designsystem.digital.gov/about/releases/) removes the `usa-layout-grid` dependency from the `usa-footer` package, which reduces that package size but breaks things.

We fix this by re-importing it manually.

### CODE REVIEW

**Review by jimlerza — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "@fortawesome/react-fontawesome": "^0.2.0",
     "@joi/date": "^2.1.0",
     "@opensearch-project/opensearch": "^2.4.0",
-    "@uswds/uswds": "3.7.0",
+    "@uswds/uswds": "^3.7.1",
```
Comment (TomElliottFlexion) [inline package.json]: I think we should leave this locked (to 3.7.1). I have zero faith in USWDS maintaining correct semver now

**Review by TomElliottFlexion — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (jasonfu9966) [reply package.json]: I'm half with ya. I also no longer have faith in USDWS maintaining correct semver, but I'm a little disappointed this got through both manual and automated testing.

Comment (TomElliottFlexion) [reply package.json]: Splitting hairs here, but it only got through manual testing _because_ it wasn't version locked

Comment (TomElliottFlexion) [reply package.json]: But I do agree!

**Review by ujahio — APPROVED** [review]

**Review by mmarcotte — APPROVED** [review]

**Pull request merged**